mclagsyncd/mclaglink.cpp

```diff
@@ -0,0 +1,814 @@
+/* Copyright(c) 2016-2019 Nephos.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms and conditions of the GNU General Public License,
+ * version 2, as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, see <http://www.gnu.org/licenses/>.
+ *
+ * The full GNU General Public License is included in this distribution in
+ * the file called "COPYING".
+ *
+ *  Maintainer: Jim Jiang from nephos
+ */
+
+#include <string.h>
+#include <errno.h>
+#include <system_error>
+#include <stdlib.h>
+#include <iostream>
+#include "logger.h"
+#include "netmsg.h"
+#include "netdispatcher.h"
+#include "swss/notificationproducer.h"
+#include "mclagsyncd/mclaglink.h"
+#include "mclagsyncd/mclag.h"
+#include <set>
+#include <algorithm>
+
+using namespace swss;
+using namespace std;
+
+vector<string> string_split(const string& s, const string& delim)
```

Comment:
please use the `tokenize` function in swss-common repository

Reply:
Sure, done